packages/firestore/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 # Unreleased
 - [changed] Firestore now recovers more quickly after network connectivity
```

Comment:
This item was released as part of 1.3.3. Could you create that entry and move it there?